fava/application.py

```diff
@@ -375,6 +375,7 @@ def url_for_source(**kwargs):
                 url_for_source=url_for_source,
                 api=g.api,
                 operating_currencies=g.api.options['operating_currency'],
+                num_currencies=len(g.api.options['operating_currency']),
```

Comment:
Please tell me how.  Every time I tried to use len() inside the template it got very angry at me.